x-pack/legacy/plugins/apm/server/routes/ui_indices.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
What about moving this to `x-pack/legacy/plugins/apm/server/routes/settings/ui_indices.ts`. 
Similarly `x-pack/legacy/plugins/apm/server/routes/settings.ts` should be moved to `x-pack/legacy/plugins/apm/server/routes/settings/agent_configuration.ts`